resources/views/PageDesign/Partials/Head.twig

```diff
@@ -44,31 +44,9 @@
 
 <script>
     App = {
```

Comment:
Können wir das in dem Step vllt. aus dem Header vor das Javascript am ende des Bodys verschieben?